users/utils.py

```diff
@@ -0,0 +1,23 @@
+import jwt
+
+from django.conf import settings
+from django.http import JsonResponse
+
+from .models import User
+
+def login_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
```

Comment:
inner function의 인자로 `*args`, `**kwargs`로 받는 이유는 무엇인가요?

```suggestion
    def wrapper(self, request, *args, **kwargs):
```

Reply:
확장성때문에 쓴다고만 알고있고, 왜 쓰는지 정확히 모르겠습니다. 써도 안써도 상관없어서 일단 써놓았습니다